README.md

```diff
@@ -9,12 +9,13 @@
 Repository to describe, develop, document and test the QualityOnDemand API family
 
 ## Scope
-* Service APIs for “Quality on Demand” (see APIBacklog.md)  
+
+* Service APIs for “Quality on Demand” (see APIBacklog.md)
 * It provides the customer with the ability to:  
-  * set quality for a mobile connection (e.g. required latency, jitter, bit rate)  
+  * set quality for access network connections (e.g. required latency, jitter, bit rate)  
```

Comment:
```suggestion
  * set quality for access network connections (e.g. mobile device connection or fixed access towards a home gateway)
  * set the quality to QoS profiles provided by the network operator (with e.g. certain latency, jitter and bit rate parameters)  
```

some more clarity, that it isn't possible to require arbitrary parameters.

Reply:
I'll incorporate this feedback, but would like to use something other than `towards a home gateway,` since this could be interpreted as only downstream flows.  What about `fixed access between a home gateway and the service providers gateway router`?

Reply:
@RandyLevensalor get your point and totally fine with your proposal